src/opower/utilities/coned.py

```diff
@@ -0,0 +1,105 @@
+"""condEdison (ConEd)."""
+
+from pyotp import TOTP
+
+import aiohttp
+
+from ..const import USER_AGENT
+from ..exceptions import InvalidAuth
+from .base import UtilityBase
+
+LOGIN_BASE = "https://www.coned.com/sitecore/api/ssc/ConEdWeb-Foundation-Login-Areas-LoginAPI/User/0"
+LOGIN_HEADERS = {
+    "User-Agent": USER_AGENT,
+    "Referer": "https://www.coned.com/",
+}
+RETURN_URL = "/en/accounts-billing/my-account/energy-use"
+

```

Comment:
Run black. It needs 2 empty lines after constants. Sorry I just copied lint and formatting requirements from Home Assistant. I need to add them as a pre-commit.